src/modules/GameConstants.ts

```diff
@@ -360,6 +360,16 @@ export enum Currency {
     contestToken,
 }
 
+export enum CurrencyRate {
+    money = 200_000.0,
+    questPoint = 60.0,
+    dungeonToken = 15_000.0,
+    diamond = 1.0,
+    farmPoint = 60.0,
+    battlePoint = 10.0,
+//    contestToken = 60.0,
```

Comment:
Could you update these to just use standard numbers, just for code consistency:
```suggestion
    money = 200000,
    questPoint = 60,
    dungeonToken = 15000,
    diamond = 1,
    farmPoint = 60,
    battlePoint = 10,
//    contestToken = 60,
```
_(same thing with the `.d.ts` file too please)_

Reply:
Changed it, should i keep the switch numerical or do you rather want the string version?

Reply:
Should probably use the enums rather than magic numbers

Reply:
If you make this a Record<Currency, number> you won't need the switch case at all - can just lookup the value directly by currency

Reply:
Thanks for the advice combined both